x-pack/plugins/uptime/server/rest_api/uptime_route_wrapper.ts

```diff
@@ -13,7 +13,7 @@ export const uptimeRouteWrapper: UMKibanaRouteWrapper = (uptimeRoute) => ({
     tags: ['access:uptime-read', ...(uptimeRoute?.writeAccess ? ['access:uptime-write'] : [])],
   },
   handler: async (context, request, response) => {
-    const { callAsCurrentUser: callES } = context.core.elasticsearch.legacy.client;
+    const callES = context.core.elasticsearch.client.asCurrentUser;
```

Comment:
This is a wrapper function that we pass all the REST handlers through; the intention is the referencing of the es client in each handler's params is abstracted away. Changing the reference here will cause all the handlers to pass the new client to their `request` functions, which is Uptime parlance for Elasticsearch interaction.